dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -749,21 +766,39 @@ class updatedns
                 break;
             case 'cloudflare':
             case 'cloudflare-v6':
+			case 'cloudflare-token':
+			case 'cloudflare-token-v6':
                 $baseUrl = 'https://api.cloudflare.com/client/v4';
                 $fqdn = str_replace(' ', '', $this->_dnsHost);
                 $recordType = ($this->_useIPv6) ? 'AAAA' : 'A';
+                $ttlData = intval($this->_dnsTTL);
```

Comment:
could be simplified (including block below), e.g. 

`$ttlData = intval($this->_dnsTTL) < 1 ? 1 : intval($this->_dnsTTL);`

omitting intval() will likely lead to the same result by the way